src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/dft_ref.cl

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+typedef float2 cfloat;
+#if 0
```

Comment:
I propose either to add comments on why we are using macroses instead of methods with correspond benchmarks or just leave inline methods.

Reply:
i'd like to leave one implementation after gathering feedback

Reply:
functions require some macro help to support fp16 in addition to fp32, i've dropped functions